js/menu.js

```diff
@@ -1,18 +1,44 @@
-$('.show-menu-button').on('click', function() {
+/**
+ * showMenu() displays menu in smaller screens
```

Comment:
In web development, responsiveness has 4-5 types of screens:  mobile, tablet, small screens, large screens and larger screens. Often mobile, tablet and small screens are treated as smaller screens. The maximum width for smaller screen devices vary from 932px to  991px. Bootstraps have these things in their documentation. We are following semantic's standard, hence smaller screens have max width of 932px. It's already there in semantic's documentation.

Reference: [Semantic](https://semantic-ui.com/elements/container.html) | [Bootstrap](https://getbootstrap.com/docs/4.0/layout/overview/)